### PR TITLE
Update all third-party component versions and related screens - New PR

### PR DESCRIPTION
**Pull request by sunithasomanna**
Closed the old PR related to these changes because of additional feedback from the QA team and moving it to this sprint.

- Update the versions of the following components in the Getting Started Readme
   Kubespray
   etcd
   docker
   K8S
   Ubuntu 
   Redfish Schema and Spec
   kube-master 
   kube-node 
   Helm charts 
   Keepalived
   Stakater/Reloader 
   linux-headers
   docker image versions of K8S and ODIMRA services
- Renamed the following:
   kube-master - kube_control_plane
   kube-node - kube_node
- Updated screenshots of the sample output pods and services. 
- Updated team's additional feedback from Dev and QA teams

### CODE REVIEW

**Review by irappa-pattar — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

**Review by shivacharanms — COMMENTED** [review]
K8s docker images and odimra images screenshots to be updated

**Comment by sunithasomanna** [comment]
> K8s docker images and odimra images screenshots to be updated

Done!

File: README.md [hunk 1]
```diff
@@ -362,20 +365,27 @@ The following table lists the software components and their versions that are co
...
+   |quay.io/calico/kube-controllers| v3.19.2 |quay.io/calico/kube-controllers.tar |
+   |k8s.gcr.io/k8s-dns-node-cache| 1.15.13 |k8s.gcr.io/k8s-dns-node-cache.tar |
+   |k8s.gcr.io/dns/k8s-dns-node-cache|1.17.1 |k8s.gcr.io_k8s-dns-node-cache.tar |
+   |k8s.gcr.io/pause|3.4.1 |k8s.gcr.io_pause.tar |
    |nginx|1.19 |nginx.tar |
-   |quay.io/coreos/etcd|v3.4.3 |quay.io_coreos_etcd.tar |
+   |k8s.gcr.io/coredns/coredns|v1.8.0 |k8s.gcr.io/coredns/coredns.tar |
+   |quay.io/coreos/etcd|v3.4.13 |quay.io/coreos/etcd.tar |
+   |k8s.gcr.io/cpa/cluster-proportional-autoscaler-amd64|1.8.3 |k8s.gcr.io/cpa/cluster-proportional-autoscaler-amd64.tar |
+   |calico/node|v3.15.1 |calico/node.tar |
```
Comment (sukhwinder20) [inline README.md]: Old kubernetes images needs to be removed

File: README.md [hunk 2]
```diff
@@ -1253,7 +1264,7 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
 8. Save the URP Docker image on the deployment node at `~/plugins/urplugin`.
 
       ```
-     docker save urplugin:2.0 -o ~/plugins/urplugin/urplugin.tar
+     docker save urplugin:1.0 -o ~/plugins/urplugin/urplugin.tar
```
Comment (sukhwinder20) [inline README.md]: urplugin version is 3.0 in above docker image list

File: README.md [hunk 3]
```diff
@@ -2784,7 +2795,7 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
 7. Save the GRF plugin Docker image on the deployment node at `~/plugins/grfplugin`.
 
     ```
-    docker save grfplugin:2.0 -o ~/plugins/grfplugin/grfplugin.tar
+    docker save grfplugin:1.0 -o ~/plugins/grfplugin/grfplugin.tar
```
Comment (sukhwinder20) [inline README.md]: grfplugin version is 3.0

File: README.md (hunk 1 above)
Comment (sunithasomanna) [reply README.md]: removed

File: README.md (hunk 2 above)
Comment (sunithasomanna) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (sunithasomanna) [reply README.md]: done

File: README.md [hunk 4]
```diff
@@ -362,20 +365,22 @@ The following table lists the software components and their versions that are co
...
-   |lachlanevenson/k8s-helm|v3.2.3 |lachlanevenson_k8s-helm.tar |
+   |k8s.gcr.io/kube-apiserver|v1.21.5 |k8s.gcr.io_kube-apiserver.tar |
+   |k8s.gcr.io/kube-controller-manager|v1.21.5 |k8s.gcr.io_kube-controller-manager.tar |
+   |k8s.gcr.io/kube-proxy| v1.21.5 |k8s.gcr.io_kube-proxy.tar |
+   |k8s.gcr.io/kube-scheduler| v1.21.5 |k8s.gcr.io_kube-scheduler.tar |
+   |quay.io/calico/node| v3.19.2 |quay.io/calico/node.tar |
+   |quay.io/calico/pod2daemon-flexvol| v3.19.2 |quay.io/calico/pod2daemon-flexvol.tar |
+   |quay.io/calico/cni| v3.19.2 |quay.io/calico/cni.tar |
+   |quay.io/calico/kube-controllers| v3.19.2 |quay.io/calico/kube-controllers.tar |
+   |k8s.gcr.io/k8s-dns-node-cache| 1.15.13 |k8s.gcr.io/k8s-dns-node-cache.tar |
```
Comment (sukhwinder20) [inline README.md]: this is old image (k8s.gcr.io/k8s-dns-node-cache| 1.15.13 |k8s.gcr.io/k8s-dns-node-cache.tar) |

**Review by sukhwinder20 — CHANGES_REQUESTED** [review]
Remove old kubernetes image

File: README.md (hunk 4 above)
Comment (sunithasomanna) [reply README.md]: done

**Review by rohitkumar-ro — APPROVED** [review]

**Review by rohitkumar-ro — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -448,29 +452,30 @@ The following table lists the software components and their versions that are co
     ```
     docker save -o <image_name.tar> <image_name>:<version>
     ```
-    Example: `docker save -o api.tar api:1.0`
+    Example: `docker save -o api.tar api:2.0`
```
Comment (sukhwinder20) [inline README.md]: api version is 3

**Review by sukhwinder20 — CHANGES_REQUESTED** [review]
Api version is 3.0

File: README.md (hunk 5 above)
Comment (sunithasomanna) [reply README.md]: done

**Review by sukhwinder20 — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**